### PR TITLE
Example demonstrating Kanister integration with MySQL

### PR DESCRIPTION
**Pull request by PrasadG193**
## Change Overview

This PR is related to: https://github.com/kanisterio/kanister/pull/218
Move stable mysql example to `examples/stable/mysql`

## Pull request type

Please check the type of change your PR introduces:
- [ ] Work in Progress
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Trival/Minor
- [ ] Bugfix
- [ ] Feature
- [ ] Documentation

## Issues

- #XXX

## Test Plan

- [ ] Manual
- [ ] Unit test
- [ ] E2E

### CODE REVIEW

File: examples/stable/mysql/README.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+```bash
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+$ helm repo update
+
+$ helm install stable/mysql -n my-release --namespace mysql-test \
+    --set mysqlRootPassword='asd#45@mysqlEXAMPLE' \
+    --set persistence.size=10Gi
+```
+
+The command deploys MySQL on the Kubernetes cluster in the default configuration.
```
Comment (SupriyaKasten) [inline examples/stable/mysql/README.md]:
```suggestion
The command deploys an instance MySQL in `mysql-test` namespace on Kubernetes cluster with default configurations.
```

File: examples/stable/mysql/README.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+By default a random password will be generated for the root user. If you'd like to set your own password change the mysqlRootPassword
```
Comment (SupriyaKasten) [inline examples/stable/mysql/README.md]:
```suggestion
By default a random password will be generated for the root user. For setting your own password, use the `mysqlRootPassword` param as shown above.
```

File: examples/stable/mysql/README.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+in the values.yaml.
```
Comment (SupriyaKasten) [inline examples/stable/mysql/README.md]:
```suggestion
```

File: examples/stable/mysql/README.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+You can retrieve your root password by running the following command. Make sure to replace [YOUR_RELEASE_NAME]:
+
+    printf $(printf '\%o' `kubectl get secret [YOUR_RELEASE_NAME]-mysql -o jsonpath="{.data.mysql-root-password[*]}"`)
+
+> **Tip**: List all releases using `helm list`
+
+## Integrating with Kanister
+
+If you have deployed postgresql application with name other than `my-release` and namespace other than `mysql-test`, you need to modify the commands used below to use the correct name and namespace
```
Comment (SupriyaKasten) [inline examples/stable/mysql/README.md]:
```suggestion
If you have deployed MySQL application with name other than `my-release` and namespace other than `mysql-test`, you need to modify the commands(backup, restore and delete) used below to use the correct name and namespace
```

File: examples/stable/mysql/README.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+    --set persistence.size=10Gi
+```
+
+The command deploys an instance MySQL in `mysql-test` namespace on Kubernetes cluster with default configurations.
+
+By default a random password will be generated for the root user. For setting your own password, use the `mysqlRootPassword` param as shown above.
+
+You can retrieve your root password by running the following command. Make sure to replace [YOUR_RELEASE_NAME]:
+
+    printf $(printf '\%o' `kubectl get secret [YOUR_RELEASE_NAME]-mysql -o jsonpath="{.data.mysql-root-password[*]}"`)
```
Comment (tdmanv) [inline examples/stable/mysql/README.md]: Let's use base64 decode. 
```suggestion
   `kubectl get secret my-release-mysql -o jsonpath="{.data.mysql-root-password[*]}" | base64 --decode`
```

File: examples/stable/mysql/kanister/mysql-blueprint.yaml [hunk 6]
```diff
@@ -0,0 +1,73 @@
+apiVersion: cr.kanister.io/v1alpha1
```
Comment (tdmanv) [inline examples/stable/mysql/kanister/mysql-blueprint.yaml]: Let's remove `kanister` from the path.

File: examples/stable/mysql/README.md [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+
+If you have deployed MySQL application with name other than `my-release` and namespace other than `mysql-test`, you need to modify the commands(backup, restore and delete) used below to use the correct name and namespace
+
+### Create Profile
+Create Profile CR if not created already
+
+```bash
+$ kanctl create profile s3compliant --access-key 'AKIAIOSFODNN7EXAMPLE' \
+	--secret-key 'wJalrXUtnFEMI%K7MDENG%bPxRfiCYEXAMPLEKEY' \
+	--bucket <s3-bucket-name> --region ap-south-1 \
```
Comment (tdmanv) [inline examples/stable/mysql/README.md]: Let's be consistent with mixing string values and `<param>` style params

File: examples/stable/mysql/README.md [hunk 8]
```diff
@@ -0,0 +1,246 @@
...
+Once MySQL is running, you can populate it with some data. Let's add a table called "pets" to a test database:
+
+```bash
+# Connect to MySQL by running a shell inside MySQL's pod
+$ kubectl exec -ti $(kubectl get pods -n mysql-test --selector=app=my-release-mysql -o=jsonpath='{.items[0].metadata.name}') -n mysql-test -- bash
+
+# From inside the shell, use the mysql CLI to insert some data into the test database
+# Create "test" db
+
+$ mysql --user=root --password=asd#45@mysqlEXAMPLE
```
Comment (tdmanv) [inline examples/stable/mysql/README.md]: [minor]
```suggestion
$ mysql --user=root --password='asd#45@mysqlEXAMPLE'
```

File: examples/stable/mysql/README.md [hunk 9]
```diff
@@ -0,0 +1,246 @@
...
+1 row in set (0.00 sec)
+
+```
+
+
+## Troubleshooting
+
+If you run into any issues with the above commands, you can check the logs of the controller using:
+
+```bash
```
Comment (tdmanv) [inline examples/stable/mysql/README.md]: let's also describe the blueprint to get events

File: examples/stable/mysql/README.md [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+
+
+## Cleanup
+
+### Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
```
Comment (tdmanv) [inline examples/stable/mysql/README.md]:
```suggestion
$ helm delete my-release --purge
```

File: examples/stable/mysql/README.md [hunk 11]
```diff
@@ -0,0 +1,246 @@
...
+
+## Introduction
+
+This chart bootstraps a single node MySQL deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.6+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+- Kanister version 0.21.0 installed
```
Comment (tdmanv) [inline examples/stable/mysql/README.md]:
```suggestion
- Kanister controller version 0.21.0 installed in your cluster
```

File: examples/stable/mysql/kanister/mysql-blueprint.yaml [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+kind: Blueprint
+metadata:
+  name: mysql-blueprint
+actions:
+  backup:
+    type: Deployment
+    outputArtifacts:
+      mysqlCloudDump:
+        keyValue:
+          path: '/mysql-backups/{{ .Deployment.Namespace }}/{{ .Deployment.Name }}/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}/dump.sql.gz'
```
Comment (tdmanv) [inline examples/stable/mysql/kanister/mysql-blueprint.yaml]: Let's use `kando output` to ensure a consistent artifact string

File: examples/stable/mysql/kanister/mysql-blueprint.yaml (hunk 6 above)
Comment (PrasadG193) [reply examples/stable/mysql/kanister/mysql-blueprint.yaml]: @tdmanv I didn't get you. If we remove `kanister` from apiVersion, it will not work.

**Pull request merged**